docs/components/databases.rst

```diff
@@ -108,13 +108,14 @@ How to use the :code:`mysql` or :code:`mariadb` format?
 Known limitations with MySQL and MariaDB
 ----------------------------------------
 
-Do to a default setting that prevent users from creating stored functions that 
-will cause unsafe events to be written into the binary log, you will need to 
-contact your database manger (maybe your self) and ask them to
+MySQL's default settings prevent users from creating stored functions that 
+can cause unsafe events to be written into the binary log. You will need to 
+contact your database manager (maybe your self) and ask them to or do this 
+your self.
```

Comment:
**_Suggestion_**:

```rst
MySQL and MariaDB's default settings prevent ordinary users from creating 
stored functions which can cause unsafe events to be written into the 
binary log.

You will need to contact your database manager (maybe your self) and ask 
them to set this value (1) or better, ask them to import the SQL script into 
the database directly.

If you are running your own database server, we recommend you to import 
the suited SQL script directly into the database, as binary_logging is used 
for clustering and turning this value on, might cause you issues later on, as 
you have lowered the servers security level of trust.
```

Reply:
```
You will need to contact your database manager (maybe your self) and ask 
them to set this value (ad: 1) or even better, ask them to import the SQL script 
into the database directly.

If you are running your own database server, we recommend you to import 
the equivalent SQL script directly into the database, as log_bin is used 
for ex. clustering and turning this value on, might cause you issues later on, as 
you have lowered the servers security level of trust.
```

Reply:
we? who are we? I will suggest this in stead:

```
If you are running your own database server, the recommendation is you import 
the equivalent SQL script directly into the database, as log_bin is used 
for ex. clustering. Turning this value into `1`, might cause security issues later on, 
as you have lowered the servers default security level of trust.
```